config/rbac/role.yaml

```diff
@@ -105,14 +104,6 @@ rules:
   - patch
   - update
   - watch
-- apiGroups:
-  - ""
-  resources:
-  - nodes/status
-  verbs:
-  - get
-  - patch
-  - update
```

Comment:
Why are these removed? Updating extended resources requires this but I'm not sure if we support those.

Reply:
Yeah, I think we definitely need these as we have support for `ResourceLabels`